Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -89,6 +89,20 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
+    autobump_file_path = cask.tap.path/Tap::HOMEBREW_TAP_AUTOBUMP_FILE
+    if cask.tap.core_cask_tap? &&
+       autobump_file_path.exist? &&
+       autobump_file_path.readlines(chomp: true).include?(cask.token) &&
+       ENV.fetch("HOMEBREW_TEST_BOT_AUTOBUMP", nil).blank?
```

Comment:
Having read #16664 and Homebrew/homebrew-core#163023 more carefully, I think we also need to remove the line below before setting `HOMEBREW_TEST_BOT_AUTOBUMP` in the autobump action:

https://github.com/Homebrew/brew/blob/977ac20b2b491a8df2e28a95541775d0b4aa20fa/Library/Homebrew/utils/github.rb#L573

I'm saying "remove" because it's currently not used and doesn't seem to be useful according to what @Bo98 said in Homebrew/homebrew-core#163023.

(Sorry there've been so many iterations of reviews -- should be my final comment.)

Reply:
Yes

Reply:
I opened a PR to rip that out. It looks like there is now no way to skip that check.

- https://github.com/Homebrew/brew/pull/16781